lib/store/dashboard/transaction_filter_store.dart

```diff
@@ -1,21 +1,20 @@
 import 'package:mobx/mobx.dart';
 import 'package:cw_core/transaction_direction.dart';
 import 'package:cake_wallet/view_model/dashboard/transaction_list_item.dart';
+import 'package:cake_wallet/view_model/dashboard/filter_item.dart';
+import 'package:cake_wallet/generated/i18n.dart';
 
 part 'transaction_filter_store.g.dart';
 
 class TransactionFilterStore = TransactionFilterStoreBase
     with _$TransactionFilterStore;
 
 abstract class TransactionFilterStoreBase with Store {
-  TransactionFilterStoreBase(
-      {this.displayIncoming = true, this.displayOutgoing = true});
+  TransactionFilterStoreBase();
 
-  @observable
-  bool displayIncoming;
-
-  @observable
-  bool displayOutgoing;
+  Observable<bool> displayAll = Observable(true);
+  Observable<bool> displayIncoming = Observable(true);
+  Observable<bool> displayOutgoing = Observable(true);
```

Comment:
same, annotation would be better, instead of having to access it with `.value`

Reply:
also, I would suggest making those private with getters only, so they won't get mistakenly modified from outside